external_search.py

```diff
@@ -17,12 +17,25 @@ class ExternalSearchIndex(object):
     work_document_type = 'work-type'
     __client = None
 
+    CURRENT_ALIAS_SUFFIX = '-current'
+    VERSION_RE = re.compile('-v([0-9]+)$')
+
+    @classmethod
+    def reset(cls):
+        """Resets the __client object to None so a new configuration
+        can be applied during object initialization.
+
+        This method is only intended for use in testing.
+        """
+        cls.__client = None
+
     def __init__(self, url=None, works_index=None):
     
         self.log = logging.getLogger("External search index")
 
         # By default, assume that there is no search index.
         self.works_index = None
+        from_configuration = False
```

Comment:
There's also the case where you provide works-v2 and it exists already but has no alias. That shouldn't happen for any circ managers set up once we have this branch merged, but if it did I would think it should create the alias.